server.js

```diff
@@ -5768,6 +5768,48 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Jetbrains Plugins repository integration
+camp.route(/^\/jetbrainsplugins\/([^\/]+)\/(d)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var pluginId = match[1];
+  var type = match[2];
+  var format = match[3];
+  var badgeData = getBadgeData('jetbrains plugin', data);
+  var url = 'https://plugins.jetbrains.com/plugins/list?pluginId=' + pluginId;
+
+  request(url, function(err, res, buffer) {
+    if (err) {
+      badgeData.text[1] = 'inaccessible';
```

Comment:
You can handle status code 404 here, this will give users better feedback:

    if (res.statusCode === 404) {
        badgeData.text[1] = 'plugin not found';
    } else {
        badgeData.text[1] = 'inaccessible';
    }